packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/EditSpecimen/index.js

```diff
@@ -142,6 +153,10 @@ export default compose(
     allFetchedKey: 'featureTypesFetched',
     resource: 'featureType',
   }),
+  createEnsureAllItemsFetched({
+    allFetchedKey: 'allTypeFetched',
```

Comment:
no need to specify this key unless you use it (and want a custom name for it), so remove this